templates/configure_ag.j2

```diff
@@ -73,16 +73,16 @@ BEGIN
 {# In the first loop, add primaries at the end and witness at the beginning #}
 {% for item in ansible_play_hosts_all %}
 {%   if hostvars[item]['mssql_ha_replica_type'] == 'primary' %}
-{{ ag_replicas_primary_last.append(item) -}}
+{%     set _ = ag_replicas_primary_last.append(item) %}
```

Comment:
ok - note that earlier versions of ansible 2.19 did not allow the use of mutator methods such as `append`, `update`, etc. but I guess they have relaxed that restriction.